tests/api/machine.test.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Nanocloud turns any traditional software into a cloud solution, without
+ * changing or redeveloping existing source code.
+ *
+ * Copyright (C) 2016 Nanocloud Software
+ *
+ * This file is part of Nanocloud.
+ *
+ * Nanocloud is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Nanocloud is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General
+ * Public License
+ * along with this program.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+/* globals sails, AccessToken, User, Machine */
+
+var nano = require('./lib/nanotest');
+
+module.exports = function() {
+  describe('machine', function() {
+    it('Should return user machines', function(done) {
+      User.create({
+        'first-name': 'Test',
+        'last-name': 'Test',
+        password: 'tests',
+        email: 'test@test.com',
+        isAdmin: false,
+        'expiration-date': null,
+      })
+        .then(() => {
+          return Machine.create({
+            name: 'essai1',
+            type: 'manual',
+            ip: '1.1.1.1',
+            username: 'Administrator',
+            password: 'admin'
+          },
+          {
+            name: 'essai2',
+            type: 'manual',
+            ip: '2.2.2.2',
+            username: 'Administrator',
+            password: 'admin'
+          });
+        })
+        .then(() => {
+          return nano.request(sails.hooks.http.app)
```

Comment:
It is preferable to create user's token by directly calling `AccessToken.create`.
Here is an example: https://github.com/Nanocloud/nanocloud/blob/faa23db05f0931db340c63a975641b0e28bb39a0/tests/api/apps.test.js#L210